streams/src/main/java/org/apache/kafka/streams/kstream/KStreamBuilder.java

```diff
@@ -75,6 +91,27 @@ public KStreamBuilder() {
         return new KStreamImpl<>(this, name, Collections.singleton(name));
     }
 
+
+    /**
+     * Create a {@link KStream} instance from the specified Pattern.
+     * <p>
+     * If multiple topics match the Pattern there are nor ordering guaranteed for records from different topics.
+     *
+     * @param keySerde  key serde used to read this source {@link KStream},
```

Comment:
Mention that this serde is used for all topics, hence all possibly matched topics should have the same message key-value data format.